core/src/main/java/com/sequenceiq/cloudbreak/service/AbstractWorkspaceAwareResourceService.java

```diff
@@ -52,10 +54,14 @@ public T create(T resource, Workspace workspace, User user) {
             prepareCreation(resource);
             setWorkspace(resource, user, workspace);
             return repository().save(resource);
-        } catch (DataIntegrityViolationException e) {
-            String message = String.format("%s already exists with name '%s' in workspace %s",
-                    resource().getShortName(), resource.getName(), resource.getWorkspace().getName());
-            throw new BadRequestException(message, e);
+        } catch (AccessDeniedException e) {
```

Comment:
Is it intentional not to remap the DataIntegrityViolationException anymore? 
Is it handled by an outer layer?